UI.AutomationTests/appsettings.json

```diff
@@ -41,7 +41,7 @@
       "Environment": "Development",
       "BrowserType": "Chrome",
       "ConnectionString": "",
-      "AdminUrl": "https://vh-admin-web.dev.platform.hmcts.net/reform-login",
+      "AdminUrl": "https://localhost:5400/reform-login",
```

Comment:
Can we change this back to dev for consistency with the VideoUrl?

Reply:
yes! forgot to revert before push it up